source/common/access_log/access_log_manager_impl.cc

```diff
@@ -16,15 +16,24 @@ void AccessLogManagerImpl::reopen() {
   }
 }
 
-AccessLogFileSharedPtr AccessLogManagerImpl::createAccessLog(const std::string& file_name) {
-  if (access_logs_.count(file_name)) {
-    return access_logs_[file_name];
+AccessLogFileSharedPtr AccessLogManagerImpl::createAccessLog(const std::string& file_name_arg) {
+  const std::string* file_name = &file_name_arg;
+#ifdef WIN32
+  // Preserve the expected behavior of specifying path: /dev/null on Windows
```

Comment:
Do windows devs actually expect this? Should we just have windows users specify NUL?

Reply:
right now yeah, this is the case, Windows users do have to use `NUL` but there are a lot of hard coded test fixtures that we would need to template `NUL` into instead of `/dev/null`; this was an attempt to not have to do that, but ultimately that might be the more correct/better solution (though a bigger change)

Reply:
Up to you, but I would at least leave a TODO trail here.

Reply:
My thinking on this was that "Path: /dev/null" has a well-understood meaning. It is worth an additional doc patch submission to note that this is "supported on windows" for the particular exception cases that we accept this exceptional path.

Reply:
We're also considering doing the proper replacement of `/dev/null` again at the test level since there already seems to be a framework set up to do this in all the yaml test assets we just noticed: https://github.com/envoyproxy/envoy/blob/eb9e13626830ad46a9c754035db3fa81b32a8f82/test/test_common/environment.cc#L206

(usage: https://github.com/envoyproxy/envoy/blob/eb9e13626830ad46a9c754035db3fa81b32a8f82/test/common/secret/secret_manager_impl_test.cc#L64)